control-plane/csi-driver/src/bin/controller/server.rs

```diff
@@ -35,9 +34,10 @@ impl Connected for UnixStream {
 }
 
 #[derive(Clone, Debug)]
-pub struct UdsConnectInfo {
-    pub peer_addr: Option<Arc<tokio::net::unix::SocketAddr>>,
-    pub peer_cred: Option<tokio::net::unix::UCred>,
+#[allow(unused)]
```

Comment:
Do we need this?

Reply:
Yes, before fields were public so linter didn't complain...
Marking it unused so it's clearer that they are not used, probably can remove them but I'm not sure if it's safe to do it or not.